scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -179,8 +188,9 @@ object ScroogeSBT extends AutoPlugin {
       streams,
       scroogeThriftSources,
       scroogeThriftOutputFolder,
-      scroogeThriftIncludes
-    ) map { (out, sources, outputDir, inc) =>
+      scroogeThriftIncludes,
+      scroogeGenerated
```

Comment:
Looks like this is initialized to the empty list so scrooge will always regenerate thrift, i.e. this patch breaks the dirty check.

The problem is that it doesn't know what the output files will be.

One workaround is to generate output in a separate directory, so no other files are there.